src/views/Landing.vue

```diff
@@ -266,18 +264,20 @@ export default {
                 return;
             }
 
-            this.videoDevice =
-                (this.mediaDevices.find(
+            this.videoDevice = (
+                this.mediaDevices.find(
                     d =>
                         d.label ===
                         this.localUser?.stream?.getVideoTracks()[0]?.label
-                )?.deviceId || this.videoInputDevices[0]).deviceId;
-            this.audioDevice =
-                (this.mediaDevices.find(
+                )?.deviceId || this.videoInputDevices[0]
+            ).deviceId;
+            this.audioDevice = (
+                this.mediaDevices.find(
                     d =>
                         d.label ===
                         this.localUser?.stream?.getAudioTracks()[0]?.label
-                )?.deviceId || this.audioInputDevices[0]).deviceId;
+                )?.deviceId || this.audioInputDevices[0]
+            ).deviceId;
```

Comment:
Just formatting

Reply:
Instead of looping there just use the ones from the store, videoDeviceId and audioDeviceId